Examples/Framework/CMakeLists.txt

```diff
@@ -34,6 +38,11 @@ target_compile_definitions(
   ActsExamplesFramework
   PRIVATE BOOST_FILESYSTEM_NO_DEPRECATED)
 
+if (ACTS_BUILD_PLUGIN_ONNX)
+  target_sources(ActsExamplesFramework PRIVATE src/EventData/NeuralCalibrator.cpp)
+  target_link_libraries(ActsExamplesFramework PRIVATE ActsPluginOnnx)
+endif()
+
```

Comment:
`Framework/ML ` might be a much better idea, I agree. I don't have any strong feeling about this but I think it is something we should discuss now instead of later.

Reply:
Good idea ! I will still review the rest of the PR in the mean time

Reply:
Was this discussed at the last dev meeting, @Corentin-Allaire @benjaminhuth ?

Reply:
Done